sci-geosciences/grass/grass-7.8.3.ebuild

```diff
@@ -0,0 +1,273 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_REQ_USE="sqlite"  # bug 572440
+WANT_AUTOCONF="2.1"
+WX_GTK_VER=3.0
+
+inherit autotools desktop eapi7-ver python-single-r1 toolchain-funcs wxwidgets xdg flag-o-matic
+
+MY_PM=${PN}$(ver_cut 1-2 ${PV})
+MY_PM=${MY_PM/.}
+MY_P=${P/_rc/RC}
+
+DESCRIPTION="A free GIS with raster and vector functionality, as well as 3D vizualization"
+HOMEPAGE="https://grass.osgeo.org/"
+SRC_URI="https://grass.osgeo.org/${MY_PM}/source/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0/7.8.3"
+KEYWORDS="amd64 ~ppc ~x86"
```

Comment:
Don't commit straight to stable. Repoman should warn you about that?

We [request](https://devmanual.gentoo.org/keywording/index.html) stabilisation on Bugzilla after ~30 days usually.